docs/modules/ROOT/pages/admin.adoc

```diff
@@ -102,6 +102,31 @@ You can modify your multisig settings by creating _custom action_ proposals to e
 
 image::defender-admin-add-owner.png[Defender Admin Add Owner]
 
+[[timelocks]]
+== Timelocks
+
+=== Creating timelocked proposals
+
+Defender Admin supports timelocked admin proposals via the https://docs.openzeppelin.com/contracts/4.x/access-control#using_timelockcontroller[TimelockController contract provided by the OpenZeppelin Contracts library].
+
+To execute a timelocked proposal, you need a `TimelockController` contract to have enough permissions to execute it. Once proper permissions are in place, just create a proposal as you normally would, ticking the `Timelock` checkbox in the _Execution strategy_ section. Then enter your timelock's address and choose the minimum delay between the proposal's approval and its execution. 
```

Comment:
> To execute a timelocked proposal, you need a `TimelockController` contract to have enough permissions to execute it. Once proper permissions are in place,

This part is a bit confusing. Are you referring to execution or creation? Whichever the case, let's make it explicit what permissions are needed (like you are doing on the Managing section, which is super clear!). Perhaps we can actually start by explaining the setup? Like "you need a multisig, that's `proposer` in a timelock, that has rights over your contract".

Reply:
Gave it another try. I'll probably release to prod before you get a chance to review this, so if it looks fine feel free to merge so the docs update.